### ISSUE TITLE
Webinstaller UX Issues

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/install-from-web-server#74]

**Comment by Llewellynvdm** [comment]
Hi @coolcat-creations the UI is found here: https://github.com/joomla/joomla-cms/blob/4.2-dev/plugins/installer/webinstaller/tmpl/default.php#L44 and much of the flow of how the install process work is manage with this: https://github.com/joomla/joomla-cms/blob/4.2-dev/plugins/installer/webinstaller/webinstaller.php

What can I do to help you? I have read over the issue you made... so my question is, should we/me write the code to change the UI?

**Comment by JohnG1985** [comment]
There are several potential ways to fix the issue described. Here are a few possible solutions:

Simplify the download process by making the download button more prominent and easier to find. This could be done by placing the download button at the top of the page, or by highlighting it in a different color to make it stand out from the rest of the page.

Provide clear instructions on how to download and install the extension. This could be done by adding a brief tutorial or step-by-step guide at the top of the page, or by providing a link to a separate page with detailed instructions.

Add a "Download" link next to the extension's title on the overview page. This would allow users to easily access the download page without having to click on the description first.

Make the download process more streamlined by removing unnecessary steps. For example, the page could automatically take the user to the download page when they click on the extension's title or description, or the page could automatically start the download process when the user clicks on the download button.

Overall, the goal should be to make the download process as simple and intuitive as possible, so that even inexperienced users can easily find and install the extension without any frustration.

**Comment by coolcat-creations** [comment]
@JohnG1985 
You mean the install button right ? But I agree in general to what you wrote.

**Comment by uglyeoin** [comment]
I tried to find an install button on Engage Box.  I couldn't find one.  Does that mean we can't install this way?  In which case doesn't this lead to a confusing/pointless user experience?  It's called install from web but you cannot install from web.  Or am I being stupid?

**Comment by coolcat-creations** [comment]
I agree it's misleading, but you can't exclude the paid extensions there. But in general the UX is bad in this component I think...

**Comment by uglyeoin** [comment]
What does WordPress do?  I've never seen a situation where they didn't allow it.  Either they only list things with a paid version, or they force everyone to have a free version.  But I think simply signposting that it's a paid version would be fine and asking them to subscribe but then having some sort of a mechanism to install it this way afterwards would be useful.  Whatever makes it easier for the user.

**Comment by HLeithner** [comment]
I would not list paid/subscription only extensions in the IFW (or even in the JED)
